.github/workflows/dispatch-build-bottle.yml

```diff
@@ -29,7 +33,7 @@ env:
 jobs:
   bottle:
     runs-on: ${{github.event.inputs.runner}}
-    timeout-minutes: 4320
+    timeout-minutes: ${{fromJson(github.event.inputs.timeout)}}
```

Comment:
Does it not work without fromJson?

Reply:
Not tested but given it's a string type above and it's failed on `tests.yml` for that reason: pretty sure it's needed.